clang/test/CIR/CodeGen/array.cpp

```diff
@@ -29,8 +29,15 @@ int f[5] = {1, 2};
 
 void func() {
   int arr[10];
-
   // CHECK: %[[ARR:.*]] = cir.alloca !cir.array<!s32i x 10>, !cir.ptr<!cir.array<!s32i x 10>>, ["arr"]
+
+  int e = arr[1];
```

Comment:
Can you add a test case for `arr[0]`? That's a special case in terms of resulting LLVM IR, but is still represented as an explicit ptr_stride in CIR.